sdk/eventhub/Azure.Messaging.EventHubs/src/EventProcessorClient.cs

```diff
@@ -873,30 +927,35 @@ private async Task RunAsync(CancellationToken cancellationToken)
                                                               ProcessingStoppedReason reason)
         {
             if (PartitionPumps.TryRemove(partitionId, out PartitionPump pump))
+            {
+                await StopPartitionPumpAsync(partitionId, reason, pump);
```

Comment:
I believe we're missing a `ConfigureAwait(false)` here.

Reply:
thanks!